Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/GameSettingsFragment.java

```diff
@@ -0,0 +1,99 @@
+package org.dolphinemu.dolphinemu.fragments;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.os.Environment;
+import android.preference.ListPreference;
+import android.preference.PreferenceFragment;
+
+import org.dolphinemu.dolphinemu.NativeLibrary;
+import org.dolphinemu.dolphinemu.R;
+import org.dolphinemu.dolphinemu.activities.GameDetailsActivity;
+import org.dolphinemu.dolphinemu.utils.Log;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.OutputStreamWriter;
+
+
+public class GameSettingsFragment extends PreferenceFragment implements SharedPreferences.OnSharedPreferenceChangeListener {
```

Comment:
curly brace should be on new line

Reply:
the whole file is missing the code style from Dolphin, check the other comment that says how to get the code style in AS